test/e2e/sagemaker/tests/test_endpoint.py

```diff
@@ -39,91 +41,174 @@ def sagemaker_client():
 
 
 @pytest.fixture(scope="module")
-def single_variant_xgboost_endpoint():
-    endpoint_resource_name = random_suffix_name("single-variant-endpoint", 32)
-    config1_resource_name = endpoint_resource_name + "-config"
-    model_resource_name = config1_resource_name + "-model"
+def get_name_suffix():
+    return random_suffix_name("xgboost-endpoint", 32)
```

Comment:
nit: fixtures don't need `get_` in their name unless they return a factory method